internal/gatewayapi/route.go

```diff
@@ -31,6 +31,8 @@ const (
 	// Request timeout, which is defined as Duration, specifies the upstream timeout for the route
 	// If not specified, the default is 15s
 	HTTPRequestTimeout = "15s"
+	// egPrefix is a prefix of annotation keys that are processed by Envoy Gateway
```

Comment:
unused ?

Reply:
used in `filterEGPrefix` to filter the annotations that should be picked up for metadata.

Reply:
how do we document this opt in behavior ? user guide ?